samples/bot-sso-adaptivecard/csharp/BotSsoAdaptivecard/README.md

```diff
@@ -0,0 +1,175 @@
+---
+page_type: sample
+description: Microsoft Teams Bot-SSO-Adaptivecard
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "03/01/2023 07:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-sso-adaptivecard-csharp
+---
+
+# SSO for your Adaptive Cards
+
+This sample code demonstrates how to get enable SSO authentication for your Adaptive Cards.
+
+## Interaction with app
+
+ ![Preview](Images/PreviewAppSSOCsharp.gif)
+
```

Comment:
We need to add one more section for demo manifest of C# code:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/636/files#diff-7bb681f0cc7d221ebbf0b17dacc3a1b81bf3bb250a4770a3162eaee026e62e57

Reply:
Fixed